srcpkgs/dolphin-plugins/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'dolphin-plugins'
+pkgname=dolphin-plugins
+version=20.04.2
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="extra-cmake-modules gettext qt5-qmake qt5-host-tools dolphin"
+makedepends="qt5-devel kxmlgui-devel kio-devel ktexteditor-devel dolphin"
+depends="dolphin"
+short_desc="These plugins integrate Dolphin with the revision control systems Bazaar, Mercurial and Git"
```

Comment:
Use smth shorter like
`Plugins to integrate Dolphin with various revision control systems`.